.github/workflows/snapshot.yml

```diff
@@ -45,27 +51,46 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Create snapshot versions
-        run: |
-          yarn version-packages --snapshot snapshot-release
+      - name: Create and publish snapshot release
+        uses: actions/github-script@v6
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Publish snapshots
-        id: publish_snapshots
-        run: |
-          body=$(yarn release --no-git-tags --snapshot | grep "New tag:")
-          body="${body//'%'/'%25'}"
-          body="${body//$'\n'/'%0A'}"
-          body="${body//$'\r'/'%0D'}"
-          body="${body//$'tag'/'release'}"
-          echo "::set-output name=body::$body"
-
-      - name: Create comment
-        uses: peter-evans/create-or-update-comment@v2
         with:
-          issue-number: ${{ github.event.issue.number }}
-          body: ${{ steps.publish_snapshots.outputs.body }}
+          script: |
+            const execa = require('execa')
+
+            const versionProcess = execa.command('npm run version-packages -- --snapshot snapshot-release')
```

Comment:
Should we use `yarn` just for consistency? I know they are effectively the same 🤷